rules/windows/process_creation/proc_creation_win_susp_obfuscated_ip_via_cli.yml

```
@@ -17,10 +18,31 @@ detection:
         Image|endswith:
             - '\ping.exe'
             - '\arp.exe'
-    selection_ip:
-        - CommandLine|contains: ' 0x' # For hex format
-        - CommandLine|re: ' [0-9]{7,13}' # For octal format
-    condition: all of selection*
+    selection_ip_1:
+        CommandLine|contains:
+            - '//0x'
+            - '.0x'
+            - '.00x'
+    selection_ip_2:
+        CommandLine|contains|all:
+            - 'http://%'
+            - '%2e'
+    selection_ip_3:
+        # http://81.4.31754
+        - CommandLine|re: 'https?://[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,5}'
+        # http://81.293898
+        - CommandLine|re: 'https?://[0-9]{1,3}\.[0-9]{1,8}'
+        # http://1359248394
+        - CommandLine|re: 'https?://[0-9]{1,10}'
+        # http://0121.04.0174.012
+        - CommandLine|re: 'https?://(0[0-9]{1,11}\.){3}0[0-9]{1,11}'
+        # http://012101076012
+        - CommandLine|re: 'https?://0[0-9]{1,11}'
+        # For octal format
+        - CommandLine|re: ' [0-9]{7,13}'
```

```suggestion
        - CommandLine|re: ' [0-7]{7,13}'
```
octal should work like this?